.gitignore

```diff
@@ -11,6 +11,7 @@ MANIFEST
 *.wpr
 *.wpu
 *.DS_Store
+Pipfile.lock
```

Comment:
The creator of pipenv recommended keeping the Pipfile and lockfile both in version control.

Reply:
Agreed, once I learned that you can still install newer stuff even if the lock file is there, it makes sense that you should put the lock file in VCS.